loader/CMakeLists.txt

```diff
@@ -117,6 +117,45 @@ if(WIN32)
     string(REGEX REPLACE "/RTC." "" MODIFIED_C_FLAGS_DEBUG ${MODIFIED_C_FLAGS_DEBUG})  #remove run-time error checks
 
     separate_arguments(MODIFIED_C_FLAGS_DEBUG WINDOWS_COMMAND ${MODIFIED_C_FLAGS_DEBUG})
+
+    # ~~~
+    # Setup the loader.rc flie to contain the correct info
+    # Creates 2 files, one for Release builds and one for every other build
+    # Optionally uses the BUILD_DLL_VERSIONINFO build option to allow setting the exact build version
+    # ~~~
+    if ("$CACHE{BUILD_DLL_VERSIONINFO}" STREQUAL "")
+        # default case - use 0 as the BUILDNO
+        set(LOADER_RC_VERSION "${LOADER_GENERATED_HEADER_VERSION}.0")
```

Comment:
My concern here is that if anyone builds it the number will look reasonable.  Part of the reason for the 111.222 or whatever was we would know they had an unofficial loader build.  Perhaps we can tie some of that in here?

Reply:
This was indeed one concern I had. But thinking about it, I don't see much of a reason to care, because in all but a _Release_ mode build the string '- Dev Build' will be added to the version, and this makes it pretty obvious if someone did a development build. Granted, it doesn't work for _Release_, but one could always set `BUILD_DLL_VERSIONINFO` manually if they desired.